src/dotnet/APIView/APIView/MethodAPIV.cs

```diff
@@ -45,6 +46,7 @@ public MethodAPIV(IMethodSymbol symbol)
                 this.ReturnType = symbol.ReturnType.ToString();
             }
             this.Accessibility = symbol.DeclaredAccessibility.ToString().ToLower();
+            this.Parent = symbol.ContainingType.Name;
```

Comment:
Do you use this?

Reply:
I use it to determine if a method is a constructor, and figured it might come in handy to track the declaring type's name in the future. I can also change the use of it to instead having a boolean `IsConstructor`.